### DISCUSSION TITLE
Discord ban in Russia

### DISCUSSION THREAD

**Discussion by mlp-0ne** [Cog-Creators/Red-DiscordBot#6464] [Q&A]
Hello everyone, today in Russia Discord was completely banned, if anyone knows what can be done, please help me and the Russians

**Comment by yamikaitou** [comment]
Change your VM provider to one that is not located in Russia

**Reply by mlp-0ne** [reply]
How?

**Reply by mlp-0ne** [reply]
I tried to bypass the ban via goodbyDPI, discord started working, but the voice channel is also unavailable

**Reply by yamikaitou** [reply]
You are on your own for finding a solution around a geo-block. As this repo is for a Discord Bot, my answer is related to the bot, not your client.

**Reply by mlp-0ne** [reply]
tnx
